docs/source/getting-started/imi-config-file.rst

```diff
@@ -205,8 +205,9 @@ These settings turn on/off (``true`` / ``false``) different steps for running th
    :widths: 30, 70
    :class: tight-table
 
-   * - ``RunSetup``
-     - Boolean to run the setup script (``setup_imi.sh``), including selected setup modules above.
+   * - ``DoPriorEmis``
```

Comment:
Does this generate the prior emissions for the entire period? Does this work with kalman mode?

Reply:
I set it up to run the prior (HEMCO standalone) simulation for 1 hour using the start date specified in the config file. However, it may make more sense to run for the entire period and save out the average emissions rates (default units are kg/m2/s). For example, using the emission rates for the start date doesn't necessarily make sense if running an annual inversion which includes seasonally varying emissions. 

I've made this change in commit 29f46830.